src/ios/CDVVisit.m

```diff
@@ -25,7 +25,10 @@ - (void)startMonitoring:(CDVInvokedUrlCommand*)command {
 
 #pragma mark - CLLocationDelegate
 - (void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatus:(CLAuthorizationStatus)status {
-  if (status == kCLAuthorizationStatusAuthorizedAlways) {
+  if (status == kCLAuthorizationStatusNotDetermined) {
+    // let the app continue to try authorizing
```

Comment:
I think you need to ask again here, otherwise requestAlwaysAuthorization method won't be called?